doc/GuideForCommitters.md

```diff
@@ -91,21 +91,21 @@ build, add a comment to the pull request that follows the syntax:
    Jenkins build [ all | {platform-list} ]
    ```
 Where `{platform-list}` is a comma-separated list of platforms chosen from the
-following tags:
-
-| Tag      | Platform |
-| :----    | :------- |
-| zlinux   | 64-bit Linux on Z |
-| zos      | 64-bit zOS |
-| plinux   | 64-bit Power Linux LE |
-| aix      | 64-bit AIX |
-| x32linux | 32-bit Linux |
-| xlinux   | 64-bit Linux |
-| xcrlinux | 64-bit Compressed References Linux |
-| win      | 64-bit Windows |
-| osx      | 64-bit macOS |
-| arm      | 32-bit AArch32 (cross-compile build only) |
-| aarch64  | 64-bit AArch64 (cross-compile build only) |
+following names and aliases:
+
+| Spec Name           | Alias    | Architecture   | Platform                                           |
+| :--------           | :----    | :-----------   | :-------                                           |
+| linux_arm           | arm      | ARM            | 32-bit Linux on AArch32 (cross-compile build only) |
+| linux_aarch64       | aarch64  | ARM            | 64-bit Linux on AArch64 (cross-compile build only) |
+| aix_ppc-64          | aix      | PPC            | 64-bit AIX                                         |
+| linux_ppc-64_le_gcc | plinux   | PPC            | 64-bit Linux on Power LE                           |
+| linux_riscv64_cross | riscv    | RISC-V         | 64-bit Linux on RISC-V (cross-compile build only)  |
+| linux_x86           | x32linux | x86            | 32-bit Linux on x86                                |
+| osx_x86-64          | osx      | x64            | 64-bit macOS                                       |
+| win_x86-64          | win      | x64            | 64-bit Windows                                     |
+| linux_x86-64        | xlinux   | x64            | 64-bit Linux on x64                                |
+| linux_390-64        | zlinux   | Z              | 64-bit Linux on Z                                  |
+| zos_390-64          | zos      | Z              | 64-bit zOS                                         |
```

Comment:
A (hopefully) final suggestion: list entries order by "Spec Name" (so aix would be first, etc.).

Reply:
It conflicts with @0xdaryl's earlier suggestion to sort by architecture: https://github.com/eclipse/omr/pull/6724#discussion_r983605401.

Reply:
I'm not sure I agree that architecture is the best primary sort key, but I can live with that. Within architecture, I think it would be helpful to sort by "Spec Name".

Reply:
Sorted by "Spec Name" within architecture.

Reply:
Thank you.